xml/System.Runtime.CompilerServices/ConfiguredValueTaskAwaitable`1.xml

```diff
@@ -33,7 +33,7 @@
   </Attributes>
   <Docs>
     <typeparam name="TResult">To be added.</typeparam>
```

Comment:
To be added -> The type of the result produced

Reply:
Thanks! If you press Control+G, you can add the code suggestion and I can merge it. Like this:
```suggestion
    <typeparam name="TResult">The type of the produced result.</typeparam>
```